src/MusicPlayer.ts

```diff
@@ -0,0 +1,202 @@
+        border-radius: 10px;
+        box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1);
+        padding: 12px;
+    }
+    .album-art {
+      width: 7.3vh;
+      height: 7.3vh;
+      border-radius: 5px;
+      object-fit: cover;
+      margin-right: 10px;
+    }
+    .track-info {
+      flex: 4;
+      overflow: hidden;
+    }
+    .track-info h4, .track-info p, .track-info h5 {
+      margin: 0;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+    .track-info h5 {
+    font-size: 0.8rem;
+    color: #A9A9A9;
+    }
+    .track-info p{
+    color: #A9A9A9;
+    }
+    .progress-bar {
+      width: 100%;
+      height: 5px;
+      background: #ddd;
+      position: relative;
+      margin: 5px 0;
+      border-radius: 5px;
+    }
+    .player-controls{
+        display: flex;
+        flex:10; 
+        align-items: center;
+        flex-direction:column;
+        height:90%;
+    }
+    .progress {
+      height: 100%;
+      background: #666;
+      border-radius: 5px;
+      transition: width 0.1s ease;
+    }
+    .controls {
+      display: flex;
+      align-items: center;
+      gap: 10px;
+    }
+    button {
+      border: none;
+      background: transparent;
+      cursor: pointer;
+      font-size: 18px;
+    }
+    .durations{
+    display:flex;
+    width:100%;
+    justify-content: space-between;
+    }
+    .durations span{
+    font-size:0.8rem;
+    color:#5F5F5F;
+    }
+    .album-name:{
+    font-size: 0.5em;
+    }
+  `;
+  // controllers events 
+  private _onPrevious() {
+    this.dispatchEvent(new CustomEvent('previous-track', {
+        bubbles: true,
+        composed: true
+    }));
+}
+
+private _onPlayPause() {
+    this.isPlaying = !this.isPlaying;
+    this.dispatchEvent(new CustomEvent('playback-toggle', {
+        bubbles: true,
+        composed: true,
+        detail: { isPlaying: this.isPlaying }
+    }));
+}
+
+private _onNext() {
+    this.dispatchEvent(new CustomEvent('next-track', {
+        bubbles: true,
+        composed: true
+    }));
+}
+//progress bar events
+private _isDragging = false;
+
+private _onStartDrag(e: MouseEvent) {
+    this._isDragging = true;
+    this._updateTimeFromClick(e);
+}
+
+private _onDrag(e: MouseEvent) {
+    if (!this._isDragging) return;
+    this._updateTimeFromClick(e);
+}
+
+private _onStopDrag() {
+    this._isDragging = false;
+}
+
+private _updateTimeFromClick(e: MouseEvent) {
+    const progressBar = e.currentTarget as HTMLElement;
+    const rect = progressBar.getBoundingClientRect();
+    const x = e.clientX - rect.left;
+    const width = rect.width;
+    const percentage = Math.min(Math.max(x / width, 0), 1);
+    const newTime = percentage * this.duration;
+    
+    this.currentTime = newTime;
+    this.dispatchEvent(new CustomEvent('time-update', {
+        bubbles: true,
+        composed: true,
+        detail: { currentTime: newTime }
+    }));
+}
+  render() {
+    if(this.title){
+    return html`
```

Comment:
```suggestion
    return html`
    return html`
      <link
        rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
      />
```

this will allow to use font awesome inside this component